code/src/components/MovieList.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from "react";
+import { MovieThumb } from "components/MovieThumb";
+
+import "./movieThumb.css";
+
+export const MovieList = () => {
+  const MOVIES_URL =
+    "https://api.themoviedb.org/3/movie/popular?api_key=9dbb13757543047a90651a50d540380e&language=en-US&page=1";
+  const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    fetch(MOVIES_URL)
+      .then((response) => response.json())
+      .then((json) => setMovies(json.results));
+  }, []);
+  return (
+    <section className="movie-list">
+      {movies.map((movie) => (
+      <MovieThumb  
+            key={movie.id} 
+            id={movie.id} 
+             name={movie.title}
+             background_image={movie.poster_path}
+             releaseDate={`Released ${movie.release_date}`}/>
```

Comment:
It's really good that you put the properties on a new line, but the perfectionist in me gets stuck on the fact that they are slightly differently indented. One thing you could do in VSCode is to press `option+shift+F` (since you're on a Mac, right?) which will **auto-format the code** and get rid of small things like this.